src/main/asciidoc/_chapters/hbase_mob.adoc

```diff
@@ -613,6 +613,13 @@ $ hdfs dfs -count /hbase/mobdir/data/default/some_table
 This data is spurious and may be reclaimed. You should sideline it, verify your application’s view
 of the table, and then delete it.
 
+==== MOB data is written in the normal /hbase/data directory rather than in /hbase/mobdir
```

Comment:
How about a heading "Data values over the MOB threshold show up stored in the non-MOB hfiles"